modules/conversion-center/classes/render/contact-buttons-render-base.php

```diff
@@ -219,7 +219,7 @@ protected function render_send_button(): void {
 								[ 'aria-hidden' => 'true' ]
 							);
 						?>
-						<?php echo( $send_button_text ); ?>
+						<?php Utils::print_unescaped_internal_string( $send_button_text ); ?>
```

Comment:
Are you sure this should be unescaped? It is free text input, it can be open to Stored XSS attacks.

Reply:
Also, did we use `print_unescaped_internal_string()` in other places in the widgets as well where we maybe shouldn't have? User input should always be escaped. Or is there a reason why you think these shouldn't be escaped?

Reply:
@mserino @nicoladj77  I was able to to a stored XSS attack on the Link In Bio widget for example. We've had a plethora of these security issues coming in recently, so I decided to check it here as well :)

<img width="1508" alt="Screenshot 2024-04-29 at 6 47 14 PM" src="https://github.com/elementor/elementor/assets/20996103/3f54919f-74bd-45e9-a5e6-621c38507d04">